README.md

```diff
@@ -21,3 +23,30 @@ How to use:
     3. R - Clear changes in the stroke editor
     4. N - Next character
```

Comment:
Missing alot of bindings. Also, i never heard of `N` before

Reply:
Well, `N` could be one of the following:
1. `n`
1. `j`
1. `k`

Colemak user moment :skull: